lib/cloudcmd/lib/client.js

```diff
@@ -2,17 +2,17 @@ var Util, DOM, CloudFunc, join;
 
 (function(scope, Util, DOM, CloudFunc) {
     'use strict';
-    
```

Comment:
Yarg giant whitespace diffs! That said I approve removing trailing whitespace.

Reply:
Yes I should have done that on a separate branch to avoid making the review more difficult. My bad.